Lab_1/IK-01_Smolij_Natan/reader.c

```diff
@@ -0,0 +1,8 @@
+#include "my_lib.h"
+
+int main()
+{
+	printf("Welcome!\n");
+	read_wav("Master Of Puppets.wav");
```

Comment:
Do you test open this file? Please, check< is space isn't a ESC sequence. If file not opened, try "Master\ Of\ Puppets.wav"